src/NuGet.Core/NuGet.ProjectModel/LockFile/LockFileUtilities.cs

```diff
@@ -26,5 +27,20 @@ public static LockFile GetLockFile(string lockFilePath, Common.ILogger logger)
 
             return lockFile;
         }
+
+        public static LockFile SafeGetLockFile(string lockFilePath, Common.ILogger logger)
```

Comment:
I was saying to use it everywhere, I think it's safe enough to use across restore :smile: